### ISSUE TITLE
fix alt text and figure out why images are missing

### ISSUE THREAD

**Issue by rabbah** [nimbella/docs#49]
Visit https://docs.nimbella.com/deploy-from-github and you'll see this at the end bottom of the page:

<img width="339" alt="Screen Shot 2021-02-15 at 3 44 31 PM" src="https://user-images.githubusercontent.com/4959922/107991491-b4964a00-6fa4-11eb-94c2-908d72f8ee5a.png">

There are three references to this the code base, as alt-text for images.
https://github.com/nimbella/docs/search?q=Docusaurus+with+Keytar

https://github.com/nimbella/docs/blob/877fecaa36d480b281066a8ed23d055fd37879f8/docs/key-value-storage.md#L36
https://github.com/nimbella/docs/blob/5bdc4b88c42ed39c94807dd7ed25dc466df4613f/docs/project-creating-your-own.md#L21
https://github.com/nimbella/docs/blob/5bdc4b88c42ed39c94807dd7ed25dc466df4613f/docs/deploy-from-github.md#L39

**Comment by rabbah** [comment]
Fixed by https://github.com/nimbella/docs/commit/28e97404e87801e7c47f1ad98240985bf5737cc2 https://github.com/nimbella/docs/commit/51c94af6e7639b4978ad08bac6c33083eeb8f592 and https://github.com/nimbella/docs/commit/0038e07fd4a96f4c993bedec2b64dd5a4164f3ec.
